src/openrct2/ride/Ride.h

```diff
@@ -134,297 +134,297 @@ struct Ride
+    const RideStation& getStation(StationIndex stationIndex = StationIndex::FromUnderlying(0)) const;
+    std::span<RideStation> getStations();
+    std::span<const RideStation> getStations() const;
+    StationIndex getStationIndex(const RideStation* station) const;
 
     // Returns the logical station number from the given station. Index 0 = station 1, index 1 = station 2. It accounts for gaps
     // in the station array. e.g. if only slot 0 and 2 are in use, index 2 returns 2 instead of 3.
-    StationIndex::UnderlyingType GetStationNumber(StationIndex in) const;
+    StationIndex::UnderlyingType getStationNumber(StationIndex in) const;
 
 public:
     uint16_t inversions{};
     uint16_t holes{};
-    uint8_t sheltered_eighths{};
+    uint8_t shelteredEighths{};
 
     std::unique_ptr<RideMeasurement> measurement;
 
 private:
-    void Update();
-    void UpdateQueueLength(StationIndex stationIndex);
-    ResultWithMessage CreateVehicles(const CoordsXYE& element, bool isApplying);
-    void MoveTrainsToBlockBrakes(const CoordsXYZ& firstBlockPosition, TrackElement& firstBlock);
-    money64 CalculateIncomePerHour() const;
-    void ChainQueues() const;
-    void ConstructMissingEntranceOrExit() const;
-
-    ResultWithMessage ChangeStatusDoStationChecks(StationIndex& stationIndex);
-    ResultWithMessage ChangeStatusGetStartElement(StationIndex stationIndex, CoordsXYE& trackElement);
-    ResultWithMessage ChangeStatusCheckCompleteCircuit(const CoordsXYE& trackElement);
-    ResultWithMessage ChangeStatusCheckTrackValidity(const CoordsXYE& trackElement);
-    ResultWithMessage ChangeStatusCreateVehicles(bool isApplying, const CoordsXYE& trackElement);
+    void update();
+    void updateQueueLength(StationIndex stationIndex);
+    ResultWithMessage createVehicles(const CoordsXYE& element, bool isApplying);
+    void moveTrainsToBlockBrakes(const CoordsXYZ& firstBlockPosition, TrackElement& firstBlock);
+    money64 calculateIncomePerHour() const;
+    void chainQueues() const;
+    void constructMissingEntranceOrExit() const;
+
+    ResultWithMessage changeStatusDoStationChecks(StationIndex& stationIndex);
+    ResultWithMessage changeStatusGetStartElement(StationIndex stationIndex, CoordsXYE& trackElement);
+    ResultWithMessage changeStatusCheckCompleteCircuit(const CoordsXYE& trackElement);
+    ResultWithMessage changeStatusCheckTrackValidity(const CoordsXYE& trackElement);
+    ResultWithMessage changeStatusCreateVehicles(bool isApplying, const CoordsXYE& trackElement);
 
 public:
-    bool CanBreakDown() const;
-    RideClassification GetClassification() const;
-    bool IsRide() const;
-    void Renew();
-    void Delete();
-    void Crash(uint8_t vehicleIndex);
-    void SetToDefaultInspectionInterval();
-    void SetRideEntry(ObjectEntryIndex entryIndex);
+    bool canBreakDown() const;
+    RideClassification getClassification() const;
+    bool isRide() const;
+    void renew();
+    void remove();
```

Comment:
I'm fine with `Delete` being changed to `remove`, but any particular reason?

Reply:
`delete` (lower case) seems to cause confusion in my IDE or in GCC (since it is also used in function context).

Reply:
Ah, `delete` is a keyword, so that makes sense. Thanks, good call.